bootloaders/eboot/eboot.c

```diff
@@ -35,7 +35,7 @@ int print_version(const uint32_t flash_addr)
     uint32_t fmt[2];
     fmt[0] = S('v', '%', '0', '8');
     fmt[1] = S('x', '\n', 0, 0);
-    ets_printf((const char*) fmt, ver);
+    ets_printf((const char*) &fmt, ver);
```

Comment:
fmt is already a pointer, so this change should be undone.

Reply:
Yup, missed that the arrays were already pointers. Oops. It's only cp that is affected.